validator/validator.c

```diff
@@ -2411,17 +2433,17 @@ ds_response_to_ke(struct module_qstate* qstate, struct val_qstate* vq,
 		if(!ds) {
 			log_warn("internal error: POSITIVE DS response was "
 				"missing DS.");
-			errinf(qstate, "no DS record");
+			errinf_ede(qstate, "no DS record", reason_bogus);
```

Comment:
Anything but `reason_bogus`

```suggestion
			errinf_ede(qstate, "no DS record", LDNS_EDE_DNSSEC_BOGUS);
```